### DISCUSSION TITLE
Explore anaerobic growth in C. intermedia

### DISCUSSION THREAD

**Discussion by IVANDOMENZAIN** [IVANDOMENZAIN/diverseYeasts_metabolism#12] [Ideas]
Let's explore the model capabilities for anaerobic growth. This species is reported to be typically aerobic, so a first natural approach would be to explore cellular growth in an unconstrained environment but without oxygen. If growth is observed then let's run gene essentiality analysis for this conditions and compare results with our sequence data.

**Comment by abrilvalras** [comment]
Hi! I've been running the models with no oxygen and there doesn't seem to be any growth. I'm running an FSEOF just out of curiosity, but I wanted to supplement the media with the substrate you told me about last lession, @IVANDOMENZAIN (for anaerobic growth in S. cerevisiae). Can you remind me what it was? Thanks!

**Comment by IVANDOMENZAIN** [comment]
Hi @abrilvalras! nice to see that this is moving forward. [Here](https://github.com/SysBioChalmers/yeast-GEM/blob/master/ComplementaryScripts/otherChanges/anaerobicModel.m) you can find a script for enabling uptake of the additional required components for anaerobic growth with the [yeastGEM](https://github.com/SysBioChalmers/yeast-GEM). Take a look to the exchange reactions and the nutrients that they are related to and check if they're also present in our model.

**Comment by abrilvalras** [comment]
Hi! I'm trying to run the anaerobic growth command, but this error keeps appearing - any idea if I should change or add something? I'm using the model itself as the "model" variable. I've checked whether all of the reactions and mets are there and some of them are missing (in process of adding them), but I'm not sure if the errors have anything to do with that.
![image](https://user-images.githubusercontent.com/75130047/105632877-ce38eb80-5e55-11eb-83c1-a6e6e37bd596.png)

**Comment by IVANDOMENZAIN** [comment]
Hi @abrilvalras, thanks for the update and reporting the bugs. We can solve these issues in our session of Today :)

**Comment by IVANDOMENZAIN** [comment]
@abrilvalras  [here](https://github.com/IVANDOMENZAIN/diverseYeasts_metabolism/blob/feat/explore_anaerobic_growth/code/explore_oxygen.m) you have the updated script for anaerobic growth. It seems that the model grows in unconstrained media. Next task: let's find the essential components for this anaerobic growth.

**Comment by abrilvalras** [comment]
Just found out that Cecilia already has RNA-Seq data on C. intermedia growing in different media and oxygen utilization!
https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6670/?query=candida+intermedia&exptype%5B%5D=%22sequencing+assay%22
Might use this for my applied bioinformatics project hehe.
